examples/rollup/README.md

```diff
@@ -21,8 +21,13 @@ pip3 install -r requirements.txt
 
 ```
 Run 
-yarn install --> installs all packages
-yarn start --> runs rollup to compile and starts the server
+yarn install # installs all packages
+yarn start # runs rollup to compile and starts the server
 ```
 
 3. Go to [http://localhost:4242/index.html](http://localhost:4242/index.html)
+
+Note: if you are doing local development, you can use yalc to test local changes
+
+1. yalc add @stripe/connect-js # in this sample
```

Comment:
See https://github.com/stripe/connect-js/pull/55#discussion_r1357426934, this isn't how I've tested it before, however if you are confident it works then I am peaceful :)